packages/jsx/src/index.ts

```diff
@@ -110,12 +112,14 @@ export const reatomJsx = (ctx: Ctx, DOM: DomApis = globalThis.window) => {
         styleId = styles[val] = random().toString()
         stylesheet.innerText += '[data-reatom="' + styleId + '"]{' + val + '}\n'
       }
-      /** @see https://www.measurethat.net/Benchmarks/Show/11819/0/dataset-vs-setattribute */
+      /** @see https://www.measurethat.net/Benchmarks/Show/11819 */
       element.setAttribute('data-reatom', styleId)
-    } else if (key === 'style' && typeof val === 'object') {
```

Comment:
Wow, why you removed the style props handling? Please, restore it and add additional tests for css vars, which will ensure that we will not remove this feature again later :)